library/src/main/java/com/afollestad/vvalidator/field/value/FieldValue.kt

```diff
@@ -0,0 +1,49 @@
+/**
+ * Designed and developed by Aidan Follestad (@afollestad)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.afollestad.vvalidator.field.value
+
+import androidx.annotation.IdRes
+
+/** @author Bernat Borras (@alorma) */
+open class FieldValue<T>(
```

Comment:
All of these classes need a tiny bit of documentation - partially for maintainability and partially because Codacy will complain otherwise